test/feed.test.js

```diff
@@ -172,5 +172,19 @@ describe('Post data class tests', () => {
       expect(esSearchDelete.values).toStrictEqual([]);
       expect(esSearchDelete.results).toBe(0);
     });
+
+    test('clearCache should set lastModified and etag to null', async () => {
+      let feed = new Feed(data.author, data.url, data.user, data.link, 'etag', 'lastModified');
+      feed = await Feed.byId(await Feed.create(feed));
+      // Basic check
+      expect(feed.etag).not.toBe(data.etag);
```

Comment:
Why not check for `etag` and `lastModified` directly?

Should we do this for a few feeds vs just 1?

Reply:
Will add more feeds for tests